.github/workflows/push-images.yml

```diff
@@ -0,0 +1,45 @@
+name: Push image to ECR when changes are pushed to master
+
+on:
+ push:
+   branches:
+     - master
+ workflow_dispatch:
+
+jobs:
+  prepare-build:
+    name: Prepare build for image push to ECR
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+      - name: Capture argo image versions
+        id: componentVersions
+        run: |
+          echo "::set-output name=config::$(jq -c .components versions.json)"
+      - name: Upload versions file
+        uses: actions/upload-artifact@v2
+        with:
+          name: versions
+          path: versions.json
+    outputs:
+      config: ${{ steps.componentVersions.outputs.config }}
+
+  push-image:
+    runs-on: ubuntu-20.04
+    name: Push image to ECR
+    needs: prepare-build
+    strategy:
+      matrix:
+        versions: ${{ fromJson(needs.prepare-build.outputs.config) }}
+    steps:
+      - uses: actions/checkout@v2
+      - name: Build and push argo images to ECR
+        uses: kciter/aws-ecr-action@v4
+        with:
+          access_key_id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          secret_access_key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          account_id: '008577686731'
```

Comment:
Does this need to be secret?

Reply:
No (I don't think so anyway), the account id isn't secret in all the other repos ([pghero](https://github.com/department-of-veterans-affairs/vets-api-pghero/blob/master/.github/workflows/mirror-images.yml#L44),[ loki-canary](https://github.com/department-of-veterans-affairs/vsp-infra-loki-canary/blob/main/.github/workflows/mirror-images.yaml#L39), etc). I believe it's just the account_id for ECR

Reply:
Cool just wanted to double check.